### ISSUE TITLE
Feature Request: Remove modules under image in SVG output

### ISSUE THREAD

**Issue by TheAwiteb** [erwanvivien/fast_qr#64]
Currently, the SVG output includes modules beneath the image. By removing these, the QR code will have a more polished and professional look.

**Comment by erwanvivien** [comment]
The reason I kept the modules behind the image was that since the image can be remote (not base64) it could take time to load, having modules behind makes the QRCode valid and not weird in the transition.

You tried with transparent images, is that the problem?

**Comment by TheAwiteb** [comment]
> having modules behind makes the QRCode valid and not weird in the transition.

Maybe we can make it an option? Because if I'm using a base64 image, I'd prefer to remove the background modules. However, if the image is a remote image, I would be afraid that the image might have been deleted or changed, and the QR code would look unacceptable. So it depends on how you bring the image, so I think it's good that it's an option and not there by default.

> You tried with transparent images, is that the problem?

Yes, as you can see below the Rust logo is hard to see

<img width="200" height="200" src="https://github.com/user-attachments/assets/ed2faf83-ac0f-4b4f-8f45-59b142ab87e1" />

Also with Forgejo logo, I think it's will be better if we remove the background modules

| With background modules | Without background  modules| With White Background
|--------|--------|--------|
| <img width="200" height="200" src="https://github.com/user-attachments/assets/1b1ff391-1cc3-49b5-88a9-9132863fed0e" /> | <img width="200" height="200" src="https://github.com/user-attachments/assets/8dfdcf74-80f5-4546-8da2-35d090805de1" /> |  <img width="200" height="200" src="https://github.com/user-attachments/assets/720eb3a4-fb12-454a-9cb0-2dab55d54cf9" />  |


### White background
A white background works well if your QR code background is white. If it's not (which is uncommon), you can change it to match your background color. However, if you're not using square modules, the white background is not ideal because parts of the modules will be visible behind the background, as you can see below

<img width="200" height="200" src="https://github.com/user-attachments/assets/c7b5ce25-0e28-41dc-9530-ae64729e04b3" />

**Comment by TheAwiteb** [comment]
When I zoom in on the white square background, I can see some of the modules that lie behind the background, as you can see below.

<img width="600" height="600" src="https://github.com/user-attachments/assets/ebfd6de8-dd3f-4f9b-a441-ccfcd142d73b" />

**Comment by erwanvivien** [comment]
Yep I understand the issue, It's an alignment problem of the underlying renderers (Chrome for example)

That said it's ugly and can be fixed.
I'll have to think of a better way to do Shape::Command and include color in it.
